docs/user_guide/assessment.rst

```diff
@@ -1,21 +1,109 @@
 Assessment
 ==========
 
-Metrics
--------
+Introduction
+------------
 
-.. currentmodule:: fairlearn.metrics
+The goal of fairness assessment is to answer the question: Which groups of 
+people may be disproportionately negatively impacted by an AI system and in 
+what ways?
+
+The steps of the assesment are as follows:
+1. Identify harms
+2. Identify the groups that might be harmed
+3. Quantify harms
+4. Compare quantified harms across the groups
 
-The :py:mod:`fairlearn.metrics` module provides the means to assess fairness-related
-metrics for models. This applies for any kind of model that users may already
-use, but also for models created with mitigation techniques from the
-:ref:`mitigation` section. The :ref:`dashboard` provides a visual way to
-compare metrics between models as well as compare metrics for different groups
-on a single model.
+We next examine these four steps in more detail.
 
-Ungrouped Metrics
+1. Identify harms
 ^^^^^^^^^^^^^^^^^
 
+For example, in a system for screening job applications, qualified candidates 
+that are automatically rejected experience an allocation harm. In a 
+speech-to-text transcription system, high error rates constitute harm in the 
+quality of service.
+
+2. Identify the groups that might be harmed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In most applications, we consider demographic groups including historically 
+marginalized groups (e.g., based on gender, race, ethnicity). We should also 
+consider groups that are relevant to a particular application. For example, for 
+speech-to-text transcription, groups based on the regional dialect or being a  
+native or a non-native speaker.
+
+It is also important to consider group intersections, for example, in addition
+to considering groups according to gender and groups according to race, it is 
+also important to consider their intersections (e.g., Black women, Latinx 
+nonbinary people, etc.).
+
+3. Quantify harms
```

Comment:
Should we add a note to this section that some harms might not be readily quantifiable. And also that if using a proxy for the harm, users will need to monitor whether the proxy remains useful over time?

Reply:
I added a few sentences to this effect in item 3.